dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtilsWrapper.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.utils;
+
+import lombok.Data;
+
+@Data
+public class PropertyUtilsWrapper {
```

Comment:
I don't see this class doing any enhancement, why we don't directly execute `PropertyUtils.getString`

Reply:
This is a class to facilitate unit tests. Currently, there is no way to mock `PropertyUtils` as we choose to remove `Powermock`. `Mockito.mockStatic` does not work for this class because `PropertyUtils` throws some kind of `initialization error` when mocking it. `Powermock` could use some mechanism to suppress the error. Therefore, we could choose either to use a `wrapper class` or refactor `PropertyUtils`.

Reply:
Wrapper class removed in the latest commit.